spec/data/approved_tags.json

```diff
@@ -0,0 +1,150 @@
+{
+    "approved_http_tags": [
+        "2FAServer",
+        "Accounting",
+        "ADC",
+        "Admin Panel",
+        "Administrative",
+        "Analytics",
+        "API",
+        "API Gateway",
+        "APIServer",
+        "APM",
+        "Appliance",
+        "ApplicationServer",
+        "ASGIServer",
+        "AuthenticationServer",
+        "AuthorizationServer",
+        "B2B",
+        "BaaS",
+        "Cache",
+        "Caching Proxy",
+        "CAS",
+        "CASB",
+        "CDN",
+        "CI/CD",
+        "Cloud",
+        "CMS",
+        "Collaboration",
+        "Communication",
+        "Compliance",
+        "COTS",
+        "CRM",
+        "Customer Service",
+        "DAM",
+        "Database",
+        "Development",
+        "DMS",
+        "DNSServer",
+        "Documentation",
+        "Domain Management",
+        "DRM",
+        "Drupal Plugin",
+        "E-Commerce",
+        "E-Learning",
+        "EbookServer",
+        "Editor",
+        "Education",
+        "Email",
+        "Email Encryption",
+        "Embedded",
+        "Engineering",
+        "Entertainment",
+        "Extension",
+        "Fax",
+        "Finance",
+        "Firewall",
+        "Firmware",
+        "Forum",
+        "Framework",
+        "Frontend",
+        "FTP",
+        "FTPServer",
+        "Hardware",
+        "HCM",
+        "Health",
+        "Hospitality",
+        "Hosting",
+        "HR",
+        "HTTP Compression",
+        "HTTP Router",
+        "Hypervisor",
+        "IaaS",
+        "IAM",
+        "IndexServer",
+        "Intranet",
+        "IoT",
+        "Issue Tracker",
+        "JavaScript",
+        "Joomla Plugin",
+        "Language",
+        "Library",
+        "LMS",
+        "Load Balancer",
+        "Login Panel",
+        "Logistics",
+        "Magento Plugin",
+        "Mail",
+        "MailServer",
+        "Management",
+        "Marketing",
+        "MDM",
+        "MediaServer",
+        "Monitoring",
+        "NAS",
+        "Network",
+        "Networking",
+        "News",
+        "Object Storage",
+        "ObjectServer",
+        "Organization",
+        "OS",
+        "PaaS",
+        "Package Manager",
+        "PAM",
+        "Panel",
+        "Parked",
+        "Password Manager",
+        "Payments",
+        "PIV",
+        "Platform",
+        "Printer",
+        "Productivity",
+        "Protocol",
+        "Proxy",
+        "ProxyServer",
+        "RaaS",
+        "RDBMS",
+        "Religion",
```

Comment:
??

Reply:
Can the Quran be constituted as anything else?